app/views/partials/channels-tab-access-control.html

```diff
@@ -1,8 +1,4 @@
-<div class="form-group" ng-class="{ 'has-error' : ngError.allow }">
-  <label><span>*</span> Allowed roles and clients</label>
-  <taglist tag-data="channel.allow" ><input class="form-control" placeholder="Enter a role and press enter..." type="text" ng-model="allowedRoles" typeahead="role for role in taglistClientRoleOptions | filter:$viewValue" /></taglist>
-  <div class="has-error-msg"><i class="glyphicon glyphicon-warning-sign"></i>{{validationRequiredMsg}}</div>
-</div>
+
```

Comment:
Let's rename this to **User Access** to make clear that's only for console users (maybe add a helper tooltip too explaining what to the user the functionality of this tab?)